client/post-editor/media-modal/detail/detail-file-info.jsx

```diff
@@ -35,19 +34,17 @@ class EditorMediaModalDetailFileInfo extends React.Component {
 				break;
 
 			case 'dimensions':
-				value = createFragment( {
-					width: (
+				value = (
```

Comment:
Could be array too. Not a big deal, but potential for styling or performance implications (by DOM node count) with wrappers.

Reply:
Will that work with the `' ✕ '` string? Oh well, I'll try and find out.

Reply:
>Will that work with the `' ✕ '` string? Oh well, I'll try and find out.

I believe it handles strings fine without the key.

Reply:
Yep, still shows up fine, and no console warnings. 34f16c71bcd53421052087f5239740218728af5f